### DISCUSSION TITLE
Any chance to move "select database" combobox like sequel pro?

### DISCUSSION THREAD

**Discussion by petrmikes** [Sequel-Ace/Sequel-Ace#1472] [General]
Dear Guys,
i would like to ask you that is any possibility switch / configure top line of app like "sequel pro". It is more compat like ACE.
It will be great to move combo to left and cleanum database name.

Do you have any ideas? 

You can see in pic:
<img width="1440" alt="Snímek obrazovky 2022-06-01 v 11 14 34" src="https://user-images.githubusercontent.com/19200079/171370763-38faec11-a76a-4205-a399-4c2a6e3698a6.png">
<img width="1437" alt="Snímek obrazovky 2022-06-01 v 11 14 48" src="https://user-images.githubusercontent.com/19200079/171370781-48623f01-0534-40ee-bfa4-2e1ae980ec4d.png">
s

**Comment by Kaspik** [comment]
Sequel Pro uses old macOS target so style that old versions of macOS used. There is no reason to go back to that legacy style as newer macOS versions use this one and we use system setup. :)
